README.md

```diff
@@ -193,6 +204,35 @@ You can also configure the behavior of cljfmt:
   :cljfmt {:indents ^:replace {#".*" [[:inner 0]]}}
   ```
 
+* `:align-bindings-args` -
+  a map of var symbols to arguments positions that require binding alignment
+  i.e. `{symbol #{1 2}`. Argument positions start at 0.
+  See the next section for a detailed explanation.
+
+  Unqualified symbols in the indents map will apply to any symbol with a
+  matching "name" - so `foo` would apply to both `org.me/foo` and
+  `com.them/foo`. If you want finer-grained control, you can use a fully
+  qualified symbol in the align-bindings-args map to configure binding alignment that
+  applies only to `org.me/foo`:
+
+  ```clojure
```

Comment:
Should I create a section for configuring opts to reduce reptitiveness?

Reply:
Anything that reorganizes other sections of the README should be part of a different PR.

Reply:
So this comment can be resolved? (feel free to resolve it yourself)